src/app/pages/AccountPage/Features/AccountSummary/index.tsx

```diff
@@ -34,7 +34,7 @@ export function AccountSummary(props: AccountSummaryProps) {
         <Box>
           <AddressBox address={address} />
           <Grid
-            columns={['auto', 'auto']}
+            columns={['max-content', 'auto']}
```

Comment:
| Before | After |
| --- | --- |
| ![auto](https://user-images.githubusercontent.com/3758846/167727332-fa9caab0-cff0-48d3-8dc3-5ccdc6749c28.png) |  ![max-content](https://user-images.githubusercontent.com/3758846/167727339-ae39dcdf-4e02-4909-9787-a6a7f949b47b.png) |

Reply:
(non-blocking) You don't like the extra spacing?

Reply:
Nope :) 

Also, this makes it more consistent with wallets that have some coins:

| Before | After |
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/3758846/169148517-ee12c204-beb6-4f15-9139-65b98dc1e9aa.png) | ![image](https://user-images.githubusercontent.com/3758846/169148600-dc3d3aac-cca5-4de1-81f6-977b920702d2.png)  |

Reply:
Sold.